File: Models/Match.cs
using System;

namespace Witter.Models
{
    public class Match
    {
        public int Id { get; set; }
        public Team TeamA { get; set; }
        public Team TeamB { get; set; }
        public DateTime Date { get; set; }
        public float TeamAOdds { get; set; }
        public float TeamBOdds { get; set; }
        public float DrawOdds { get; set; }
        public Score Score { get; set; } = null; 
    }
}

File: Witter-SPA/src/app/_models/match.ts
import { Team } from './team';
import { Score } from './score';

export interface Match {
  id: number;
  teamA: Team;
  teamB: Team;
  date: Date;
  teamAOdds: number;
  teamBOdds: number;
  drawOdds: number;
  score?: Score;
}
